std/os/linux/index.zig

```diff
@@ -1189,17 +1272,22 @@ pub fn accept4(fd: i32, noalias addr: *sockaddr, noalias len: *socklen_t, flags:
 }
 
 pub fn fstat(fd: i32, stat_buf: *Stat) usize {
-    return syscall2(SYS_fstat, @intCast(usize, fd), @ptrToInt(stat_buf));
+    return fstatat(fd, c"", stat_buf, AT_EMPTY_PATH);
```

Comment:
I don't think this is really important, but, is there a benefit to not calling fstat? The previous version of this function does slightly less work, since it has to set up 2 fewer parameters, and it supports older kernels. (Same question for e.g. `epoll_pwait`.

Reply:
Drive-by comment: arm64 doesn't have an epoll_wait() syscall, only epoll_pwait(). libc emulates epoll_wait() as epoll_pwait() with a null sigmask, just like this PR does.

(Arm64 does have fstat() though, so no reason I can see to use fstatat().)

Reply:
arm64 also has execve and execveat